src/app/(public)/public-bundles/components/PublicBundle.tsx

```diff
@@ -80,7 +80,10 @@ export const PublicBundle = ({ bundleId, language }: PublicBundleProps) => {
   }
 
   const importMetadata = bundle.importMetadata as importMetadata;
-  const courseName = importMetadata.full_course_name;
+  let courseName = importMetadata.full_course_name;
```

Comment:
In general, in typescript, a more functional approach is preferred (even if not strictly necessary). So, eg, something like:

```
const languageDescription = language !== 'en' && languages[language] ? ... : ...
const coursName = ` .... `
```